CONTRIBUTING.rst

```diff
@@ -29,19 +29,19 @@ Non-Goals:
 Development
 -----------
 
-For many tasks, it is okay to just develop using a single installed python version. But if you need to test/debug the project in multiple python versions, you need to install those version::
```

Comment:
The duplicate colon is not a typo in reStructuredText, e.g. see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks.

Reply:
ah sorry! I didn't know about that. Will revert them.